### PR TITLE
ipyleaflet notebook improvements + simple ipympl example for voila

### PR DESCRIPTION
**Pull request by martinRenou**

### CODE REVIEW

File: notebooks/10.04-ipyleaflet.ipynb [hunk 1]
```diff
@@ -438,9 +604,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "widgets-tutorial",
+   "display_name": "Python 3",
```
Comment (mwcraig) [inline notebooks/10.04-ipyleaflet.ipynb]:
```suggestion
   "display_name": "widgets-tutorial",
```

File: notebooks/10.04-ipyleaflet.ipynb [hunk 2]
```diff
@@ -438,9 +604,9 @@
...
    "language": "python",
-   "name": "widgets-tutorial"
+   "name": "python3"
```
Comment (mwcraig) [inline notebooks/10.04-ipyleaflet.ipynb]:
```suggestion
   "name": "widgets-tutorial"
```

File: notebooks/10.141-ipympl-voila.ipynb [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    "\n",
+    "slider.observe(update_lines, names='value')\n",
+    "\n",
+    "VBox([slider, fig1.canvas])"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
```
Comment (mwcraig) [inline notebooks/10.141-ipympl-voila.ipynb]:
```suggestion
   "display_name": "widgets-tutorial",
```

File: notebooks/10.141-ipympl-voila.ipynb [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+   "language": "python",
+   "name": "python3"
```
Comment (mwcraig) [inline notebooks/10.141-ipympl-voila.ipynb]:
```suggestion
  "display_name": "widgets-tutorial",
```

**Comment by mwcraig** [comment]
Do you want me to upload the data file and provide instructions for folks to download it if they want to during the tutorial?

**Comment by martinRenou** [comment]
Maybe we can use this url: `https://github.com/benbovy/xvelmap/raw/master/notebooks/wind-global.nc`?

**Comment by mwcraig** [comment]
Perfect!

**Comment by mwcraig** [comment]
The velocity example is really cool @martinRenou! I added download instructions, moved it closer to the end of the notebook and am merging it.

**Pull request merged**